apps/examples/security_test/seclink/sl_ss_test.c

```diff
@@ -0,0 +1,220 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <pthread.h>
+#include <tinyara/seclink.h>
+#include <tinyara/seclink_drv.h>
+#include <stress_tool/st_perf.h>
+#include "sl_test.h"
+
+#define SECLINK_SS_TEST_TRIAL SECLINK_TEST_TRIAL
+#define SECLINK_SS_TEST_LIMIT_TIME SECLINK_TEST_LIMIT
+#define SECLINK_SS_TEST_MEM_SIZE SECLINK_TEST_MEM_SIZE
+
+#define SL_TEST_MAX_SLOT_INDEX 3
+#define SL_TEST_START_INDEX 0
+#define SL_TEST_MAX_DATA 8192
+#define SL_SS_TEST_LIMIT_TIME 1000000
+#define SL_SS_TEST_TRIAL 1
+
+/*  Each storage has different size */
+static uint32_t g_size_arr[SL_TEST_MAX_SLOT_INDEX] = {SL_TEST_MAX_DATA, 300, 300};
+static sl_ctx g_hnd;
+static hal_data g_input[SL_TEST_MAX_SLOT_INDEX];
+static hal_data g_output[SL_TEST_MAX_SLOT_INDEX];
+static unsigned char *g_ss_data[SL_TEST_MAX_SLOT_INDEX] = {NULL,};
+
+/*
+ * Desc: Write data in secure storage
+ * Refered https://developer.artik.io/documentation/security-api/see-securestorage-test_8c-example.html
```

Comment:
As I know, we cannot access `developer.artik.io` anymore.
Please change or remove the address in comment.

Reply:
Even though it can't be accessed now. It would be better to comment where I get this code from.